### PR TITLE
Fix filter response content sample - (AzureCXP) fixes sample per MicrosoftDocs/azure-docs#39802

### PR DESCRIPTION
**Pull request by mike-urnun-msft**
resolves MicrosoftDocs/azure-docs#39802

### CODE REVIEW

**Comment by mike-urnun-msft** [comment]
#sign-off

**Comment by mike-urnun-msft** [comment]
Hi @miaojiang - Could you review and help move this one forward?

File: examples/Filter response content based on product name.policy.xml [hunk 1]
```diff
@@ -15,10 +15,10 @@
   <outbound>
     <base />
     <choose>
-      <when condition="@(context.Response.StatusCode == 200 && context.Product.Name.Equals("Starter="""))">
+      <when condition="@(context.Response.StatusCode == 200 && context.Product.Name.Equals("Starter"))">
         <set-body>
           @{
-            <!-- NOTE that we are not using preserveContent=true when deserializing response body stream into a JSON object since we don't intend to access it again. See details on https://docs.microsoft.com/en-us/azure/api-management/api-management-transformation-policies#SetBody -->
+            //NOTE that we are not using preserveContent=true when deserializing response body stream into a JSON object since we don't intend to access it again. See details on https://docs.microsoft.com/en-us/azure/api-management/api-management-transformation-policies#SetBody
```
Comment (tomkerkhove) [inline examples/Filter response content based on product name.policy.xml]: This is not valid XML, can you use `<!-- -->` please?

Comment (tomkerkhove) [reply examples/Filter response content based on product name.policy.xml]: Thanks @mike-urnun-msft !

Comment (mike-urnun-msft) [reply examples/Filter response content based on product name.policy.xml]: @tomkerkhove sure, updated!

**Comment by mike-urnun-msft** [comment]
Thanks @tomkerkhove!

**Comment by tomkerkhove** [comment]
Thank YOU!

**Pull request merged**